lib/slack-rtmapi.rb

```diff
@@ -1,14 +1,20 @@
-require_relative "slack-rtmapi/client"
-require_relative "slack-rtmapi/version"
+# encoding: utf-8
+
+require_relative 'slack-rtmapi/client'
+require_relative 'slack-rtmapi/version'
 
 require 'json'
 require 'net/http'
 
-module SlackRTM
-
+#
+# SlackRTM
+#
+class SlackRTM
+  attr_writer :client
+  attr_accessor :data
```

Comment:
data is a name too broad, not specific enough. Please rename it to something more meaningful